Segments/Blackburrow.mapproj

```diff
@@ -269807,6 +269826,193 @@ return yima;
       <script name="OnIncomingPlayer" enabled="false">
         <block><![CDATA[]]></block>
         <block><![CDATA[
+]]></block>
+        <block><![CDATA[]]></block>
+      </script>
+    </entity>
+    <entity name="BossNoteableLevel18InsaneWiz">
+      <script name="OnSpawn" enabled="true">
+        <block><![CDATA[]]></block>
+        <block><![CDATA[
+var insaneWiz = new Humanoid()
+    {
+        Name = "Insane Froglok Wizard", Alignment = Alignment.Chaotic,
+        MaxHealth = 4500, Health = 4500,
+        BaseDodge = 40,
+        Body = 453,
+        CanSwim = true,
+    
+        Experience = 250000,
+        
+        BasePenetration = ShieldPenetration.Heavy,
+        HideDetection = 45,
+
+        Immunity  =  CreatureImmunity.Poison,
+        
+        
+        VisibilityDistance = 3,
+          
+    };
+    
+    insaneWiz.Stats[EntityStat.FireProtection].Base = 100;
+    insaneWiz.Stats[EntityStat.IceProtection].Base = 75;
+    insaneWiz.Stats[EntityStat.LightningResistance].Base = 75;
+    insaneWiz.AddImmunity(typeof(WhirlwindSpell));
+    insaneWiz.AddImmunity(typeof(PoisonCloudSpell));
```

Comment:
No need to include the PC immunity anymore since it does not work. Poison Immunity + BlindFearProtection you already have, so it essentially takes care of it. 

Additionally not sure if Lava has anything weird with it as you have set the fire protection to 100. I know DB spells are on a different scale.